utils/session/index.ts

```diff
@@ -6,7 +6,7 @@ export const sessionOptions: SessionOptions = {
   password: process.env.IRON_SESSION_PWD as string,
   cookieName: 'annuaire-entreprises-user-session',
   cookieOptions: {
-    secure: process.env.NODE_ENV === 'production',
+    secure: process.env.NODE_ENV === 'production' && process.env.CI !== 'true',
```

Comment:
Note : is this the proper way to deactivate secure on CI? Or should we use a custom env variable?